src/helpers/units.ts

```diff
@@ -101,17 +101,16 @@ export function centsToDollars(cents: Cents): number {
 export function dollarsToCents(dollars: number): Cents {
   return Math.ceil(dollars * 100);
 }
-
-export function parseStartDate(startDate: string): Date | "NOW" | null {
+export function parseStartDateOrNow(startDate?: string): Date | "NOW" {
+  if (!startDate) return "NOW";
   const nowRe = /\b(?:"|')?[nN][oO][wW](?:"|')?\b/;
-  if (nowRe.test(startDate)) {
-    return "NOW";
-  }
-
+  if (nowRe.test(startDate)) return "NOW";
   const chronoDate = chrono.parseDate(startDate);
-  if (!chronoDate) {
-    return null;
-  }
+  return chronoDate ?? "NOW";
+}
 
-  return chronoDate;
+export function parseStartDate(startDate?: string | Date): Date {
+  if (startDate instanceof Date) return startDate;
+  const result = parseStartDateOrNow(startDate);
+  return result === "NOW" ? new Date() : result;
```

Comment:
In an ideal world this would be called like `coerceStartDateToDate` but this is a pedantic nit